ee/tables/tablehelpers/disclaim_darwin.go

```diff
@@ -0,0 +1,37 @@
+//go:build darwin
+// +build darwin
+
+package tablehelpers
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os/exec"
+
+	"github.com/kolide/launcher/ee/allowedcmd"
+)
+
+// Disclaimed (darwin only) replaces the cmd path and args for use via
+// our rundisclaimed subcommand
+func Disclaimed(ctx context.Context, disclaimCmdName string) ExecOps {
```

Comment:
Hrm. It feels awkward to use `disclaimCmdName` here, and not `cmd.Path` inside the anonymous function. But I assume it's like this because there's no other way to get the right arg for `allowedcmd` inside disclaim. Probably the simplest tradeoff.